### DISCUSSION TITLE
Bad request error in promitor scraper

### DISCUSSION THREAD

**Discussion by krnayak2601** [tomkerkhove/promitor#1939] [Q&A] [answered]
Hi Team,

We are using promitor 2.1.0 version for metric collection of azure managed service. We were able to successfully get data for eventhub through auto discovery but today got below error in scaper:
[14:10:01 FTL] BadRequest: Metric: ActiveConnections does not support requested dimension combination: entityname, supported ones are: , TraceId: {aa84515              4-3c51-41c0-b276-6a97f85158af}
[14:10:01 FTL] BadRequest: Metric: ActiveConnections does not support requested dimension combination: entityname, supported ones are: , TraceId: {69c9298              2-f826-4795-a6d5-756499e263a7}

Can you please help to identify what this might be related to?

**Comment by tomkerkhove** [comment]
That is odd, are there any other logs than this? Can you share your config please?

You might want to try using v2.5 or so to ensure this is still an issue.

**Comment by krnayak2601** [comment]
Hi @tomkerkhove ,

Below are the config:
DiscoveryConfig:
version: v1
azureLandscape:
  tenantId: ########
  subscriptions:
  - ########
  cloud: Global
resourceDiscoveryGroups:
- name: event-hubs-landscape
  type: EventHubs


azureMetadata:
  subscriptionId: ########
  tenantId: ########
  resourceGroupName: ########
metricDefaults:
  aggregation:
    interval: 00:05:00
  scraping:
    schedule: 0 * * ? * *
metrics:
- azureMetricConfiguration:
    aggregation:
      type: Average
    metricName: ActiveConnections
  description: "Total Active Connections for Microsoft.EventHub."
  name: ActiveConnections
  resourceType: EventHubs
  resourceDiscoveryGroups:
  - name: event-hubs-landscape
version: v1


We are planning to upgrade scraper and discovery in production

**Comment by krnayak2601** [comment, marked as answer]
The versions is not updated yet
Same config worked without promitor discovery...
azureMetadata:
subscriptionId: #############
tenantId: ############
resourceGroupName: ############
metricDefaults:
aggregation:
interval: 00:05:00
scraping:
schedule: 0 * * ? * *
metrics:

name: Active_Connections
description: "Total Active Connections for Microsoft EventHub"
resourceType: Generic
azureMetricConfiguration:
metricName: ActiveConnections
aggregation:
type: Total
resources:
resourceUri: Microsoft.EventHub/namespaces/######
[13:21:04 INF] Found value 0 for metric Active_Connections with aggregation interval 00:05:00

Good for now. I'll try using discovery once the version is updated :)
